app/src/main/java/com/nowsopt/spotify/data/service/HomeMainService.kt

```diff
@@ -0,0 +1,18 @@
+package com.nowsopt.spotify.data.service
+
+import com.nowsopt.spotify.data.model.BaseResponse
+import com.nowsopt.spotify.data.model.response.HitSongs
+import com.nowsopt.spotify.data.model.response.Playlists
+import com.nowsopt.spotify.data.model.response.PopularArtists
+import retrofit2.http.GET
+
+interface HomeMainService {
+    @GET("v1/api/members/me/playlists/recommend")
+    suspend fun getRecommendMix(): BaseResponse<Playlists>
+
+    @GET("v1/api/songs/popular")
+    suspend fun getTodayHitSong(): BaseResponse<HitSongs>
```

Comment:
```suggestion
    suspend fun getTodayHitSong(): BaseResponse<List<HitSong>>
```
이런식으로 제네릭을 리스트로 써주시면 됩니다